<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg_start</class>
 <widget class="QDialog" name="dlg_start">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Start Simulation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="lbl_text">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#ff0000;&quot;&gt;The aircraft in the simulation scenario may have changed.&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#ff0000;&quot;&gt;What do you want to do ?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btn_upgrade">
       <property name="text">
        <string>Upgrade the aircraft of the scenario</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_run">
       <property name="text">
        <string>Run the simulation without changing the aircraft</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="text">
        <string>Cancel the ATN simulation</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
